src/components/Organizations.tsx

```diff
@@ -60,7 +61,11 @@ export default function Organizations() {
                     <Table.Tr>
                         <Table.Th></Table.Th>
                         <Table.Th>Name</Table.Th>
-                        <Protect permission="org:sys_memberships:manage">
+                        <Protect
+                            permission="org:sys_memberships:manage"
+                            fallback={<AccessDeniedAlert />}
+                        >
+                            <Table.Th>Save</Table.Th>
```

Comment:
was adding the Save column intentional?  Also this one shouldn't have an Alert added, if they don't have `manage` access then the controls are hidden but they don't need to know there's additional functionality they're missing out on.

Reply:
it was not intentional, i think it happened when I merged main. got it